webpack.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js$/,
+        test: /\.jsx?$/,
```

Comment:
Yup makes sense. `test.jsx` files should still be excluded though right? We don't want to bundle those with the rest of the js and add weight to the file size.